fetch.bs

```diff
@@ -2133,6 +2133,19 @@ Unless stated otherwise, it is false.
 
 <p class=note>This flag is for exclusive use by HTML's render-blocking mechanism. [[!HTML]]
 
+<p>A <a for=/>request</a> has an associated boolean <dfn export for=request>https-upgrade</dfn>.
+Unless stated otherwise, it is false.
+
+<p class=note>This flag is for exclusive use by https-upgrades algorithm.
```

Comment:
This should probably directly reference the actual algorithm(s) or simply state "HTTPS upgrading". Same below.

Reply:
Done